Backend/Controllers/LevelController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Backend.Logic;
+using Backend.Models.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backend.Controllers
+{
+    [ApiController]
+    [Route("Levels")]
+    public class LevelController : Controller
+    {
+        private readonly LevelLogic _logic;
+        private readonly UserLevelLogic _userLevelLogic;
+        public LevelController(LevelLogic logic, UserLevelLogic userLevelLogic)
+        {
+            _logic = logic;
+            _userLevelLogic = userLevelLogic;
+        }
+
+        [HttpGet]
+        [Route("{Id}")]
+        public IActionResult GetLevelBySurveyId(int Id)
```

Comment:
Be consistent with naming conventions, either CamelCase or lowerCamelCase, but not both